src/pull_request.ts

```diff
@@ -32,6 +32,10 @@ export async function handlePullRequest() {
 
   const octokit = initOctokit(config.githubToken);
 
+  if (shouldIgnorePullRequest(pull_request)) {
+    return;
+  }
```

Comment:
Consider adding a more detailed log message before returning to provide better context about why the PR review is being skipped. This would improve observability and debugging. For example:
```typescript
if (shouldIgnorePullRequest(pull_request)) {
  info(`Skipping PR review due to ignore tag in description: ${pull_request.body}`);
  return;
}
```